### PR TITLE
Add libcupti.so path to Docker.gpu

### PR DESCRIPTION
**Pull request by thesuperzapper**
More info in #7207

### CODE REVIEW

File: tensorflow/tools/docker/Dockerfile.gpu [hunk 1]
```diff
@@ -58,6 +58,9 @@ COPY notebooks /notebooks
 # We just add a little wrapper script.
 COPY run_jupyter.sh /
 
+# We depend on libcupti.so
```
Comment (caisq) [inline tensorflow/tools/docker/Dockerfile.gpu]: I don't think this comment is necessary.

**Review by caisq — CHANGES_REQUESTED** [review]

**Review by thesuperzapper — COMMENTED** [review]
It just seems a bit out of the blue without context. You think we should remove it, or change it?

**Comment by caisq** [comment]
If you want to leave the comment in, I'm fine with that, but please revise the sentence to be more specific about what "we" refers to (i.e., TensorFlow).

**Comment by thesuperzapper** [comment]
How about, "For CUDA profiling, TensorFlow requires CUPTI."

File: tensorflow/tools/docker/Dockerfile.gpu [hunk 2]
```diff
@@ -58,6 +58,9 @@ COPY notebooks /notebooks
...
+ENV LD_LIBRARY_PATH /usr/local/cuda/extras/CUPTI/lib64:$LD_LIBRARY_PATH
```
Comment (gunan) [inline tensorflow/tools/docker/Dockerfile.gpu]: Are we sure libcupti is installed in the docker image?
Maybe this should be upstreamed into nvidia docker?

Comment (thesuperzapper) [inline tensorflow/tools/docker/Dockerfile.gpu]: Yea, it's installed, when you manually change the environment variable, it works. 
See #7207

**Comment by caisq** [comment]
@thesuperzapper Sounds good to me.

**Comment by rmlarsen** [comment]
@tensorflow-jenkins test this please

**Comment by rmlarsen** [comment]
@caisq can you approve this?

**Comment by caisq** [comment]
@rmlarsen,  @thesuperzapper said he would update the comment line. Let's wait for him on that?

**Comment by rmlarsen** [comment]
@caisq SGTM

**Comment by thesuperzapper** [comment]
Done, sorry about the delay, time-zones and such.

**Review by caisq — APPROVED** [review]
Thanks, @thesuperzapper

**Pull request merged**